boards/xtensa/intel_s1000_crb/CMakeLists.txt

```diff
@@ -2,3 +2,11 @@ if(CONFIG_PINMUX_INTEL_S1000)
   zephyr_library()
   zephyr_library_sources(pinmux.c)
 endif()
+
+set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
+  COMMAND ${PYTHON_EXECUTABLE} ${BOARD_DIR}/support/create_board_img.py
+  -i ${PROJECT_BINARY_DIR}/zephyr.bin
+  -o ${PROJECT_BINARY_DIR}/zephyr_${BOARD}.bin
+  -l 0xbe000000
```

Comment:
The load address is hardcoded, we need a way to override it from an application (specifically the bootloader app).

Reply:
well.. this has to be same as "SRAM_BASE" defined in soc/xtensa/intel_s1000/memory.h coz the zephyr image generated for s1000 is not relocatable. So, somehow, i need to pass SRAM_BASE (or something similar) as an argument instead of hard-coding. Do you agree with this?

Reply:
Yes agreed. One option might to be take `zephyr.elf` as an input to the python script and parse the start address using `xt-readelf`.

Reply:
Done.

Reply:
I see that you are leveraging pyelftools. pyelftools supports x86/x86_64 (and recently support ARM?), and does not explicitly support Xtensa. Because the usage is very simple (extract the start address from the header), it might be OK. Maybe we should just watch this, if you can add a console print of the address that is extracted from elf file, it would help detecting any wrong behavior with particular version(s) of tool/development host combination.

Reply:
I added the print now anyway. Have a look.